requirements.txt

```diff
@@ -1,8 +1,23 @@
 bleach==3.0.2
-Markdown==3.0.1
-bleach_whitelist==0.0.9
-Django>=2.1.6
-django_mptt==0.9.1
-djangorestframework>=3.9.1
+bleach-whitelist==0.0.9
+certifi==2019.11.28
+chardet==3.0.4
+defusedxml==0.6.0
+Django==2.2
 django-allauth==0.38.0
+django-js-asset==1.2.2
+django-mptt==0.9.1
 django-rest-auth==0.9.3
+djangorestframework==3.11.0
+idna==2.8
+Markdown==3.0.1
+oauthlib==3.1.0
+Pillow==6.2.1
+python3-openid==3.1.0
+pytz==2019.3
```

Comment:
When I was developing nutrigo I tried to have only main dependencies in `requirements.txt`. Would it be possible for you to keep it that way?